src/main/java/survey/Survey.java

```diff
@@ -0,0 +1,31 @@
+package survey;
+
+import java.util.ArrayList;
+
+public class Survey {
+    private Integer id;
```

Comment:
SHouldnt this class have a main (entry point to the application)?

Reply:
No. Survey is the survey object that contains a survey. There can be multiple surveys.